conf/application.conf

```diff
@@ -32,8 +32,7 @@ bootstrap.filters.sessionId.enabled = true
 # Play Modules
 # ~~~~
 # Additional play modules can be added here
-play.modules.enabled += "com.kenshoo.play.metrics.PlayModule"
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.FrontendModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.frontend.FrontendModule"
```

Comment:
@iservport We need com.kenshoo.play.metrics.PlayModule, please check again

Reply:
It is now in all frontend projects, including this. It is already called by the FrontendModule.